policyengine_us/parameters/gov/irs/credits/estate/base.yaml

```diff
@@ -55,3 +55,4 @@ values:
   2023-01-01: 12_920_000  
   2024-01-01: 13_610_000
   2025-01-01: 13_990_000  
+  2026-01-01: 6_790_000 # use 2017 value as base and cpi_w (2026Q1 projection) for uprating
```

Comment:
```suggestion
# use 2017 value as base and cpi_w (2026Q1 projection) for uprating
  2026-01-01: 6_790_000 
```

Reply:
Could you add the calculation that you did

Reply:
sure, just found I made a mistake in calculatioin